2020q3/puppet.md

```diff
@@ -0,0 +1,34 @@
+## Puppet
+
+Link:	 [PuppetLab's FreeBSD slack channel](https://puppetcommunity.slack.com/messages/C6CK0UGB1/)
+Link:    [Choria](https://choria.io/)
+
+Contact: Puppet Team <puppet@FreeBSD.org>  
+
+Since out last status report a few years ago, the puppet@ team regularly
+updated the various Puppet ports to follow upstream releases of Puppet
+4, Puppet 5 and Puppet 6.  Puppet 4 was removed when it reached EOL.
+
+More recently, an effort was made to enhance Facter 4 so that it can be
+used as a drop-in replacement of Facter 3 on FreeBSD.  Facter 4 is a
+Ruby rewrite of Facter 3, the C++ rewrite of Facter 2 which was
+initially in Ruby.  As a consequence we have two ports for Facter:
+sysutils/facter is the C++ implementation (Facter 3) and
+sysutils/rubygems-facter is the Ruby implementation (updated from Facter
+2 to Facter 4 a few weeks ago).  The Puppet 5 and Puppet 6 ports already
+allow to choose which version of Facter to use.  Facter 4 will be the
+default version of Facter with Puppet 7 which is expected to be released
+soon.
+
+We are getting ready for adding a port for Puppet 7 as sysutils/puppet7
+when it is available, along with PuppetServer 7 (sysutils/puppetserver7)
+and PuppetDB 7 (databases/puppetdb7).
+
+Regarding orchestration, most Marionette Collective ports are deprecated
+and the last component sysutils/mcollective is expected to be deprecated
+soon (Puppetlabs removed Marionette Collective from Puppet 6 and
```

Comment:
Change "Puppetlabs" to "Puppet"

Reply:
I reworded the last sentences to avoid confusion between the company and the program.